packages/form/addon/components/cf-field/input/action-button.hbs

```diff
@@ -0,0 +1,19 @@
+{{#unless @disabled}}
+  <DocumentValidity
```

Comment:
Might want to add an `onInvalid` notification?

Reply:
No, the button will be disabled if it's invalid.